15/index.php

```diff
@@ -198,6 +209,7 @@
   <li>Core-based debugger (#5425, #5448, #5513)</li>
   <li>Keyman Developer Server (#6033,#6034,#6035,#6036)</li>
```

Comment:
```suggestion
  <li>Keyman Developer Server (#6033,#6034,#6035,#6036)</li>
  <li>Keyboard compiler now warns on inconsistent use of Caps Lock to prevent unexpected behavior of the keyboard in use (#6347)</li>
```